linetools/isgm/utils.py

```diff
@@ -419,7 +419,7 @@ def table_from_complist(complist):
     return tab
 
 
-def iontable_from_components(components, ztbl=None, NHI_obj=None):
+def iontable_from_components(components, ztbl=None, NHI_obj=None, vrange=None):
```

Comment:
I would really prefer to deprecate this method that I wrote
in a really ugly way.  Is it possible to modify table_from_complist()
to do what you want?  I'm happy to help.

Reply:
Sure I think it's possible.  Could include a switch for ion table vs. component table and handle the two separately.  The ion table will not include b values or component velocities (and their errors) but will contain summed column densities.  Also, the modified table_from_complist() should contain someway to find the proper ion in the component list.

For the record, I'm not sure what is so ugly about iontable_from_components(), which leads me to believe that I might produce something no prettier.  Let's put our heads together on it?

Reply:
Yes, let's discuss today.